src/plugins/controls/public/options_list/components/options_list_editor_options.tsx

```diff
@@ -102,8 +131,43 @@ export const OptionsListEditorOptions = ({
               if (initialInput?.existsSelected) onChange({ existsSelected: false });
             },
           }}
+          data-test-subj={'optionsListControl__hideExistsAdditionalSetting'}
         />
       </EuiFormRow>
+      <EuiFormRow>
+        <>
+          <EuiSwitch
+            label={OptionsListStrings.editor.getHideSortingTitle()}
```

Comment:
> **Warning**
> Tag docs when ready for review

I would appreciate some help with the wording for the "Hide sort" setting and with the title of the additional form that shows up when this setting is set to `false` 🙇  Here's a quick GIF of this in action:

<p align="center">
<img src="https://user-images.githubusercontent.com/8698078/202543153-44daf4cb-4d76-4bfa-a40e-64d8471f6a3f.gif"/>
</p>

So there are two pieces of copy to consider: (1) `"Allow suggestions to be sorted"` and (2) `"Suggestions sorting"`

I think (1) is already pretty straightforward - if this setting is set to `false`, this hides the sorting menu from the control popover. Perhaps `"Allow suggestions to be dynamically sorted"` might be more specific? 🤷 

The wording of (2) is more difficult - basically, if the author hides the sorting button from the popover, then they most provide a default sorting method to use for this control since the analyst is no longer able to change it dynamically - hence, why the secondary form shows up when (1) is toggled to `false`. I'm not sure if `"Default suggestion sorting"` or something along the lines of `"Sort suggestions via..."`  would reflect this better? Definitely need some help with this one

Reply:
@Heenawter Here are some suggestions:

**1.**

Allow dynamic sorting of suggestions
Allow sorting of suggestions

**2.** 

Default sort order

(Maybe you don't need to repeat the word "suggestions" because it is appears in the text for the toggle?)

**3.** 

Ignore sorting when “Show only selected” is true.

**4.** 

Select a sorting strategy >  Define the sort order

Reply:
@gchaps Thanks so much 🙇 I changed everything in 82fccc5916cdcda5a3bde9611febafc39a85ce80, and I think things are much clearer now!

> What control does "Delete control" delete?

The GIF I included is of the control editor flyout - so, if you are editing an existing control, selecting the "Delete control" option will delete the control that you are currently editing 👍